src/solidation/__main__.py

```diff
@@ -437,6 +437,11 @@ def ensure_aware(dt: datetime) -> datetime:
     return dt.replace(tzinfo=timezone.utc) if dt.tzinfo is None else dt
 
 
+def sanitize_md(s: str) -> str:
+    # Remove `[]` symbols to ensure correct markdown in the references
+    return re.sub(r'[\[\]]', '', s)
```

Comment:
Wouldn't it be better to instead escape brackets, changing `[` to `\[`?

Reply:
dang, I didn't even try but it indeed seems to work:
 - [\[gh-actions\](deps): Bump actions/setup-python from 4 to 5](https://github.com/dandi/zarr_checksum/pull/42) (253 days)
 
 so I guess we would also need to escape the `\`:
 
  - [\\\[gh-actions\](deps): Bump actions/setup-python from 4 to 5](https://github.com/dandi/zarr_checksum/pull/42) (253 days)

ok, will do... now